File: cpp/cwiczenie2/main.cpp
#include <iostream>

using namespace std;

int main(){
	int liczby[9];
	int suma = 0;
	
	for(int i=1; i<11; i++){
		cout<<"Podaj liczbe "<<i<<": ";
		cin>>liczby[i-1];
		}
	
	for(int i=0; i<10; i++){
		suma=suma+liczby[i];
		}
	
	if(suma%2 == 0){
		cout<<"Wynik jest parzysty"<<endl;
		}
	else{
		cout<<"Wynik jest nieparzysty"<<endl;
		}
	
	return 0;
}

File: python/cwiczenie2/main.py
#Deklarowanie zmiennych
tablica = [0, 0, 0]
suma = 0

#Wczytywanie zmiennych i sumowanie
for i in range(0, 3):
	tablica[i] = int(input('Podaj wartosc '+str(i+1)+': '))
	suma += tablica[i]

#Wypisanie sumy
if suma == 0:
	print('Tablica jest pusta')
else:
	print('Suma wartosci wynosi: '+str(suma))

#Sprawdzanie czy suma jest parzysta
if suma % 2 == 0:
	print('Liczba jest parzysta')
else:
	print('Liczba nie jest parzysta')
